### ISSUE TITLE
How to enter full screen without opening the verse action dialog

### ISSUE THREAD

**Issue by agrogers** [AndBible/and-bible#1906]
"For instance as a user after multiple tries, and using your app for a long time, i still dont know how to enter and exit full screen without at the same time bringing up the bookmarks dialog" #1880 

It is correct that it is not possible to enter full screen by double-taping without showing the verse action dialog. Is this a problem? It does feel wrong.

I know we can use the ... menu to go full screen. Is it possible to detect a 'two finger' fling up? If so that would be a good alternative.

**Comment by bvahedy** [comment]
Andrew, It is possible to enter and exit the full screen without bringing
up the quick action menu. Just double tap in an empty area on the screen in
between verses.

**Comment by agrogers** [comment]
Oh, yes, true.

___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)

**Comment by timbze** [comment]
Empty area is quite easy to find when you have "1 verse per line" setting turned on. Otherwise not so much

**Comment by uniqueid001** [comment]
It's also easy to find empty area with a left & right margin greater than 0.  Would it be accepted to increase the default margin?

**Comment by bvahedy** [comment]
Yes, you can set the right or left margin to 5 for a window or a workspace and it will give you enough space to double tap without any issues. But you lose a bit of real estate on the screen. If you are fine with that then go for it

**Comment by tuomas2** [comment]
I have disabled in my setup full screen by double-tapping because it is very annoying to me have it happen unintentionally. If I want fullscreen I use 3dot menu -> fullscreen.

Should we disable double-tap to fullsceen by default?

**Comment by bvahedy** [comment]
Your call, I'm fine either way.

**Comment by agrogers** [comment]
If we make the change we should be sure to popup a what's new window for
the next release. I think we have one of those don't we?

The double tap full screen bites me when using the plus button to select
multiple verses. I always tap too fast.
